src/util/PathUtil.ts

```diff
@@ -115,14 +149,16 @@ export function toCanonicalUriPath(path: string): string {
  * Decodes all components of a URI path.
```

Comment:
Should have some more comments here (and in `encodeUriPathComponents`) to specify exactly that we use these functions to convert between URIs and file paths.